graylog2-server/src/test/java/org/graylog2/migrations/V20170110150100_FixAlertConditionsMigrationTest.java

```diff
@@ -159,7 +162,7 @@ public void upgradeWhenMigrationCompleted() throws Exception {
 
         migration.upgrade();
 
-        verify(collection, never()).updateOne(any(), any());
+        verify(collection, never()).updateOne(any(), (Bson) any());
```

Comment:
You can use `any(Bson.class)` here instead of casting the value.

Reply:
Done

Reply:
Thank you!